### ISSUE TITLE
Audit can be broken with .env files

### ISSUE THREAD

**Issue by KevinHock** [Yelp/detect-secrets#206]
Thanks to @OiCMudkips  for finding this 👍 

```
detect-secrets scan test_data/config.env > teeth
detect-secrets audit teeth
```
😬  and you get
```
Secret:      1 of 1
Filename:    test_data/config.env
Secret Type: Base64 High Entropy String
----------
ERROR: Secret not found on line 1!
Try recreating your baseline to fix this issue.
----------
What would you like to do? (s)kip, (q)uit: q
Quitting...
Saving progress...
```

I remains to be seen if this is the same bug as the yaml one I mentioned at the end of issue #204, but I shall try to add a test case for that, as our `test_data/` yaml currently works fine.

**Comment by domanchi** [comment]
What does the config file look like? Could this be a duplicate of https://github.com/Yelp/detect-secrets/issues/189?

**Comment by KevinHock** [comment]
Closed as dupe of #189 👍
